cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -426,6 +426,7 @@ impl xcm_executor::Config for XcmConfig {
 	type HrmpChannelAcceptedHandler = ();
 	type HrmpChannelClosingHandler = ();
 	type XcmRecorder = PolkadotXcm;
+	type AssetConverter = ();
```

Comment:
Rococo Asset Hub should support this too.

```suggestion
	type AssetConverter = PoolAssetsConverter;
```